src/packages/@ncigdc/modern_components/GeneExpression/GeneExpressionChart.js

```diff
@@ -41,6 +42,7 @@ class GeneExpressionChart extends Component {
     };
     // this doesn't work if jquery is imported
     // in this file. ignore the eslint error
+    // about the $ symbol on the line below
```

Comment:
this can be avoided adding the global variable to the eslint config, like so:
![image](https://user-images.githubusercontent.com/2107110/87585872-b108b080-c6ad-11ea-830d-7a6ba8704a25.png)

Reply:
nice, thanks!